src/encoded/tests/helpers/test_upload_credentials.py

```diff
@@ -0,0 +1,379 @@
+# pylint: disable=protected-access
+"""
+Tests for upload_credential.py
+"""
+from unittest import TestCase
+
+import pytest  # pylint: disable=import-error, unused-import
+
+from moto import mock_sts  # pylint: disable=import-error, unused-import
+
+from encoded.helpers import UploadCredentials
+from encoded.helpers import EXTERNAL_BUCKET_STATEMENTS
+
+from . import HELPERS_DATA_PATH
+
+
+@mock_sts
+def test_external_creds():
+    '''
+    Original test from file.py for test_external_creds
+
+    I do not think the mock works here.
```

Comment:
I would be concerned if the mock isn't working.

Reply:
Going to further look into the mock.  I could use it in other tests too.

Reply:
Looks like mock is working.  I was assuming I could print the mocked object and see it was mocked.  The moto package seems to mock requests and return "real" boto objects.  

I added the moto mock to the class also.  I'll push the code up next week after resolving all the comments.